src/api/WebLink.js

```diff
@@ -28,6 +29,70 @@ class WebLink extends Item {
         const suffix: string = id ? `/${id}` : '';
         return `${this.getBaseApiUrl()}/web_links${suffix}`;
     }
+
+    /**
+     * Gets a Box weblink
+     *
+     * @param {string} id - Weblink id
+     * @param {(newItem: BoxItem) => void} successCallback - Function to call with results
+     * @param {Function} errorCallback - Function to call with errors
+     * @param {Array<String>} fields - Array of field strings
+     * @returns {Promise}
+     */
+    async getWeblink(
+        id: string,
+        successCallback: (newItem: BoxItem) => void,
+        errorCallback: Function,
+        { fields }: FetchOptions = {},
+    ): Promise<void> {
+        if (this.isDestroyed()) {
```

Comment:
Is this check needed? I don't think it's possible for the API to be destroyed as soon as it's invoked.

Reply:
Copied from File.js, trying to be consistent 😂